libraries/AP_HAL_ChibiOS/hwdef/MatekF405-Wing/hwdef.dat

```diff
@@ -199,6 +199,7 @@ define GENERATOR_ENABLED 0
 define AC_OAPATHPLANNER_ENABLED 0
 define PRECISION_LANDING 0
 define HAL_BARO_WIND_COMP_ENABLED 0
-define HAL_RUNCAM_ENABLED 0
-define HAL_SPEKTRUM_TELEM_ENABLED 0
 define HAL_SOARING_ENABLED 0
+define HAL_WITH_DISPLAY 0
+define HAL_WITH_OREO_LED 0
```

Comment:
I agree with @WickedShell this should be off by default and enabled by the Solo builds.

Reply:
Done